### DISCUSSION TITLE
Need features to focus on new,un-read discussion/reply postings

### DISCUSSION THREAD

**Discussion by dearvcc** [community/community#112509] [Discussions]
Need some kind of features to show new,un-read discussion/reply postings,because those are things we really focus on.

**Comment by gil-son** [comment]
Hello,

Click on this icon to view:

![aaaaa](https://github.com/community/community/assets/72712095/4be086e8-d19f-4756-99ac-02d313bafba1)

If I supported you, consider to mark my answer

**Reply by dearvcc** [reply]
I’m using Github Mobile App,could not find where is such button,anyway,thank you very much!
